corehq/apps/events/templates/event_attendees.html

```diff
@@ -11,17 +11,24 @@
 {% endblock %}
 
 {% block page_content %}
-{% registerurl 'paginated_attendees' domain %}
-{% registerurl 'attendees_config' domain %}
+  {% registerurl 'paginated_attendees' domain %}
+  {% registerurl 'attendees_config' domain %}
+  {% registerurl 'event_attendees' domain %}
+
+  <p>
+    {% blocktrans %}
+    Known potential attendees who can be invited to participate in Attendance
+    Tracking Events.
+    {% endblocktrans %}
+  </p>
 
-<p>
-  {% blocktrans %}
-    This page lists all the project's potential attendees who can be invited to
-    participate in Attendance Tracking Events.
-  {% endblocktrans %}
-</p>
   <div class="btn-toolbar" style="margin-bottom: 20px;">
-    <button type="button" class="btn btn-primary">
+    <button type="button"
+            class="btn btn-primary ko-template"
+            id="new-attendee-modal-trigger"
+            data-toggle="modal"
+            data-target="#new-attendee-modal"
+                    data-bind="click: initializeAttendee">
```

Comment:
Nit: Indentation should be one back to fall in line with the rest of the properties.